CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -374,10 +462,23 @@ def checkRowForMismatch(row, columnDict, fixedColumnDict, baseColumnIdx=None, ig
                     # If output value mismatch is present, style the cell with MISMATCH_FILL_STYLE
                     # If Text Formatting mismatch is present, style the cell with LESSER_MISMATCH_FILL_STYLE
                     if len(mismatchTypes) > 0:
-                        curMismatchFillStyle = LESSER_MISMATCH_FILL_STYLE_NAME
+                        curMismatchFillStyle = None
+                        mismatch_present = False
+                        lesser_mismatch_present = False
+                        lang_mismatch_present = False
```

Comment:
can this be rewritten such that these variables are not declared? it looks like that should be possible to assign `curMismatchFillStyle` directly